index.js

```diff
@@ -9,19 +9,21 @@ const Link = require('siren-parser/dist/Link');
 class Client {
   /**
    * performs an async HTTP request
-   * returns object { body, contentType }
+   * returns object { body, contentType, statusCode }
```

Comment:
I think we should actually just have them return the whole response
then we can add the response to the parsed object instead of just the statusCode
I feel like we will eventually need something else from the response as well.

Reply:
we can call it `rawResponse` or something